app/src/main/java/com/wpi/audiojournal/screen/GeneralCategoryScreen.kt

```diff
@@ -0,0 +1,82 @@
+package com.wpi.audiojournal.screen
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.Button
+import androidx.compose.material.IconButton
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import com.wpi.audiojournal.MenuItem
+import com.wpi.audiojournal.R
+
+@Composable
+fun GeneralCategoryScreen(menuItems: List<MenuItem>) {
+    //BACK BUTTON
+    Button(
+        modifier = Modifier
+            .padding(
+                horizontal = 1.dp,
+                vertical = 1.dp
+            )
+            .fillMaxSize()
+            .size(width = 10.dp, height = 8.dp),
+        shape = CircleShape,
+        onClick = { /*TODO*/ }
```

Comment:
It seems most of these screens can be abstracted into one Composable View that can be reused for each instance

Reply:
They'd just be different in implementation in `NavGraph.kt` I believe